db/seeds.rb

```diff
@@ -26,6 +26,8 @@
 FactoryBot.create_list(:event, 3, user: frosty)
 FactoryBot.create_list(:event, 3, user: santa)
 
+Question.create!(name: 'Name', question: 'When is your name?', onboarding: true)
+Question.create!(name: 'Email', question: 'When is your email address?', onboarding: true)
 Question.create!(name: 'Birthdate', question: 'When is your birthday?', onboarding: true)
```

Comment:
@krogowsk531 @aemcdonald @taylorjohnson141 should name, email, and birthdate all be questions that live in the database? Right now they will be included in the getOnboardingQuestions request but I can take them out!

Reply:
I'm thinking it might be better to separate those question into a separate query if we keep them on the backend. My thinking is if we are giving users the options to skip the onboarding questions we shouldn't even make the OnBoardingQuestions query if they skip. However we need the name, email and birthday questions in order to create a user. Which feels like the logic should be divided. It may be easier to just hard code the questions in the FE. What do you think @leahriffell @aemcdonald @krogowsk531

Reply:
Ok so either name, email, and birthdate could be hard coded on the FE _or_ we could create a separate query called something like `getCreateUserQuestions`. Just lemme know what works better for y'all!

Reply:
Per our convo this morning, removed the user sign up questions from the seeds!